tests/cpp/test_alias.cpp

```diff
@@ -1659,4 +1660,33 @@ TEST_F(AliasTest, SliceOfExpandedBroadcast) {
       executor_cache.fusion(), out_tensors, {in_tensor}, __LINE__, __FILE__);
 }
 
+TEST_F(AliasTest, AccumulateSlices) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  constexpr int n = 5;
+
+  TensorView* acc_in = makeContigConcreteTensor({1});
+  TensorView* in = makeContigConcreteTensor({n});
+  Val* i = IrBuilder::create<Val>(DataType::Index);
+  Slice s = {i, add(i, IrBuilder::create<Val>(1)), IrBuilder::create<Val>(1)};
+  TensorView* slice_out = slice(in, {s});
+  TensorView* acc_out = add(acc_in, slice_out);
+  fusion->addInput(acc_in);
+  fusion->addInput(in);
+  fusion->addInput(i);
+  fusion->addOutput(acc_out);
+  fusion->aliasOutputToInput(acc_out, acc_in, AllocationType::ReuseBuffer);
```

Comment:
This calls TransformReplay::selfReplay to replay transformations from concrete dimensions to symbolic dimensions.